### PR TITLE
Update zh-CN translation

### PR DESCRIPTION
**Pull request by auroursa**
I noticed that the Simplified Chinese localization hasn't been updated for a while, so I translated those terms. @mitian233 @RitsukiP Please help me review this PR.

### CODE REVIEW

File: src/locale/locales/zh-CN/messages.po [hunk 1]
```diff
@@ -244,7 +244,7 @@ msgstr "成人内容"
 
 #: src/components/moderation/ModerationLabelPref.tsx:114
 msgid "Adult content is disabled."
-msgstr ""
+msgstr "已停止显示成人内容"
```
Comment (RitsukiP) [inline src/locale/locales/zh-CN/messages.po:247]: 建议改为「成人内容已被禁用」

File: src/locale/locales/zh-CN/messages.po [hunk 2]
```diff
@@ -551,11 +551,11 @@ msgstr "Bluesky 不会向未登录的用户显示你的个人资料和帖子。
 
 #: src/lib/moderation/useLabelBehaviorDescription.ts:53
 msgid "Blur images"
-msgstr ""
+msgstr "模糊图片"
```
Comment (RitsukiP) [inline src/locale/locales/zh-CN/messages.po:554]: 建议改为「模糊化图片」

File: src/locale/locales/zh-CN/messages.po [hunk 3]
```diff
@@ -584,15 +584,15 @@ msgstr "来自 {0}"
...
+msgstr "来自 {0}"
 
 #: src/view/com/profile/ProfileSubpageHeader.tsx:161
 msgid "by <0/>"
 msgstr "来自 <0/>"
 
 #: src/view/com/auth/create/Policies.tsx:87
 msgid "By creating an account you agree to the {els}."
-msgstr ""
+msgstr "通过创建账户，你同意遵守 {els}。"
```
Comment (mitian233) [inline src/locale/locales/zh-CN/messages.po:595]:
```suggestion
msgstr "创建账户即默认表明你同意我们的 {els}。"
```

File: src/locale/locales/zh-CN/messages.po [hunk 4]
```diff
@@ -4005,19 +4005,19 @@ msgstr "保存图片裁切"
...
+msgstr "保存到自定义信息流"
 
 #: src/view/screens/SavedFeeds.tsx:122
 msgid "Saved Feeds"
 msgstr "已保存信息流"
 
 #: src/view/com/lightbox/Lightbox.tsx:81
 msgid "Saved to your camera roll."
-msgstr ""
+msgstr "已保存到相机相册"
```
Comment (mitian233) [inline src/locale/locales/zh-CN/messages.po:4016]:
```suggestion
msgstr "已保存到相机胶卷"
```
建议改为相机胶卷或者直接叫相册，相机相册这个叫法不太常见。

File: src/locale/locales/zh-CN/messages.po [hunk 5]
```diff
@@ -4154,7 +4154,7 @@ msgstr "选择以下一些账户进行关注"
 
 #: src/components/ReportDialog/SubmitView.tsx:135
 msgid "Select the moderation service(s) to report to"
-msgstr ""
+msgstr "选择要举报给的限制提供服务"
```
Comment (mitian233) [inline src/locale/locales/zh-CN/messages.po:4157]:
```suggestion
msgstr "你要将该条举报提交给哪位限制服务提供者？"
```
image

File: src/locale/locales/zh-CN/messages.po [hunk 6]
```diff
@@ -4789,7 +4789,7 @@ msgstr "服务条款"
 #: src/lib/moderation/useReportOptions.ts:79
 #: src/lib/moderation/useReportOptions.ts:87
 msgid "Terms used violate community standards"
-msgstr ""
+msgstr "所使用的术语违反了社区准则"
```
Comment (mitian233) [inline src/locale/locales/zh-CN/messages.po:4792]: Terms的意思需要考究一下

File: src/locale/locales/zh-CN/messages.po [hunk 7]
```diff
@@ -5325,31 +5325,31 @@ msgstr "将文本文件上传至："
...
-msgstr ""
+msgstr "从文件上传"
 
 #: src/view/com/util/UserAvatar.tsx:337
 #: src/view/com/util/UserAvatar.tsx:341
 #: src/view/com/util/UserBanner.tsx:127
 #: src/view/com/util/UserBanner.tsx:131
 msgid "Upload from Library"
-msgstr ""
+msgstr "从相册上传"
```
Comment (mitian233) [inline src/locale/locales/zh-CN/messages.po:5340]:
```suggestion
msgstr "从媒体库上传"
```
camera roll 翻译成相机胶卷
library 翻译成媒体库

**Review by mitian233 — COMMENTED** [review]
Some translations need to be reviewed

File: src/locale/locales/zh-CN/messages.po (hunk 6 above)
Comment (auroursa) [reply src/locale/locales/zh-CN/messages.po:4792]: 可能“服务条款”更好一点？其他 Terms 的翻译串也会同步修改

**Comment by mitian233** [comment]
LGTM

File: src/locale/locales/zh-CN/messages.po [hunk 8]
```diff
@@ -3191,63 +2905,43 @@ msgstr "开启可配置的语言设置"
...
 
 #: src/view/screens/Settings/index.tsx:756
 msgid "Opens modal for changing your Bluesky password"
-msgstr ""
+msgstr "开启密码修改界面"
 
 #: src/view/screens/Settings/index.tsx:718
 msgid "Opens modal for choosing a new Bluesky handle"
-msgstr ""
+msgstr "开启创建新的用户标识符界面"
```
Comment (auroursa) [inline src/locale/locales/zh-CN/messages.po:2936]:
```suggestion
msgstr "开启创建新的用户识别符界面"
```

File: src/locale/locales/zh-CN/messages.po [hunk 9]
```diff
@@ -2322,54 +2095,41 @@ msgstr "他会显示你所关注的人发布的帖子。"
...
 
 #: src/view/screens/Profile.tsx:186
 msgid "Labels"
-msgstr ""
+msgstr "标记"
 
 #: src/screens/Profile/Sections/Labels.tsx:143
 msgid "Labels are annotations on users and content. They can be used to hide, warn, and categorize the network."
-msgstr ""
+msgstr "标记是对用户和内容的注释。它们可以用于隐藏、警告和对内容进行分类。"
```
Comment (auroursa) [inline src/locale/locales/zh-CN/messages.po:2120]:
```suggestion
msgstr "标记是对特定内容及用户的提示。可以针对特定内容显示警告；默认隐藏内容，或关闭此标记功能。"
```

File: src/locale/locales/zh-CN/messages.po [hunk 10]
```diff
@@ -3069,11 +2791,11 @@ msgstr "裸露"
 
 #: src/lib/moderation/useReportOptions.ts:71
 msgid "Nudity or pornography not labeled as such"
-msgstr ""
+msgstr "未标记的裸露或色情内容"
 
 #: src/lib/moderation/useLabelBehaviorDescription.ts:11
 msgid "Off"
-msgstr ""
+msgstr "关闭"
```
Comment (auroursa) [inline src/locale/locales/zh-CN/messages.po:2798]:
```suggestion
msgstr "显示"
```

File: src/locale/locales/zh-CN/messages.po [hunk 11]
```diff
@@ -2322,54 +2095,41 @@ msgstr "他会显示你所关注的人发布的帖子。"
...
 
 #: src/view/screens/Profile.tsx:186
 msgid "Labels"
-msgstr ""
+msgstr "标记"
 
 #: src/screens/Profile/Sections/Labels.tsx:143
 msgid "Labels are annotations on users and content. They can be used to hide, warn, and categorize the network."
-msgstr ""
+msgstr "标记是对特定内容及用户的提示。可以针对特定内容显示警告；默认隐藏内容，或关闭此标记功能。"
```
Comment (auroursa) [inline src/locale/locales/zh-CN/messages.po:2120]:
```suggestion
msgstr "标记是对特定内容及用户的提示。可以针对特定内容默认隐藏内容、显示警告或直接显示。"
```

File: src/locale/locales/zh-CN/messages.po [hunk 12]
```diff
@@ -2181,23 +1979,23 @@ msgstr "若不勾选，则默认为全年龄向。"
...
+msgstr "如果你移除此列表，将无法恢复"
 
 #: src/view/com/modals/ChangePassword.tsx:148
 msgid "If you want to change your password, we will send you a code to verify that this is your account."
 msgstr "如果你想要更改密码，我们将向你发送一个验证码以验证这是你的账户。"
 
 #: src/lib/moderation/useReportOptions.ts:36
 msgid "Illegal and Urgent"
-msgstr ""
+msgstr "非法且紧急"
```
Comment (auroursa) [inline src/locale/locales/zh-CN/messages.po:1998]:
```suggestion
msgstr "违法"
```

File: src/locale/locales/zh-CN/messages.po [hunk 13]
```diff
@@ -1390,23 +1227,23 @@ msgstr "受 Apple 政策限制，显示成人内容只能在完成注册后在
 
 #: src/view/com/modals/ChangeHandle.tsx:257
 msgid "e.g. alice"
-msgstr ""
+msgstr "例如：爱丽丝"
```
Comment (auroursa) [inline src/locale/locales/zh-CN/messages.po:1230]:
```suggestion
msgstr "例如：alice"
```

**Comment by auroursa** [comment]
Looks good to me

**Review by pfrazee — APPROVED** [review]
Sweet!! Thanks yall!

**Pull request merged**